### DISCUSSION TITLE
Help with composition + channel events

### DISCUSSION THREAD

**Discussion by chimericdream** [storybookjs/storybook#27134] [Help]
### Summary

I am working on a set of Storybook instances for a monorepo, where several packages have their own independently deployable Storybook setup, and there is also a central "umbrella" Storybook that uses composition/refs to display everything at once.

I was troubleshooting some issues with themed components when I realized that events broadcast from the outer instance of Storybook are not propagated to the composed ones.

How do I set up events/handlers/refs/whatever such that a "child" Storybook instance will receive an event like `DARK_MODE_EVENT_NAME` from the `storybook_dark_mode` addon when the user updates the mode in the main "shell" Storybook?

**Comment by DiazAilan** [comment]
I think this is related to:
https://github.com/storybookjs/storybook/issues/20001
and
https://github.com/storybookjs/storybook/issues/15805

I'm currently working in a solution for our team but not having luck for the moment.
